hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -892,6 +893,9 @@ message SnapshotInfo {
   // note: shared sizes can be calculated from: referenced - exclusive
   optional bool deepCleanedDeletedDir = 19;
   optional bytes lastTransactionInfo = 20;
+  optional uint64 exclusiveDirSize = 21;
+  // snapshot exclusive size after replication
+  optional uint64 exclusiveDirReplicatedSize = 22;
```

Comment:
I'd suggest naming the variable as specific as possible since this is updated exclusively by `SnapshotDirectoryCleaningService`:

```suggestion
  optional uint64 exclusiveSizeDeltaFromSDCS = 21;
  // snapshot exclusive size after replication
  optional uint64 exclusiveReplicatedSizeDeltaFromSDCS = 22;
```

Reply:
done